secound commit

Jaciubek opened the pull request:
fixed paths, removed picture tags, blocked display of appropriate classes depending on the selected display (mobile, tablet, desktop)

Jaciubek committed:
secound commit

fixed paths, removed picture tags, blocked display of appropriate classes depending on the selected display (mobile, tablet, desktop)

Jaciubek commented:
fixed paths, removed picture tags, blocked display of appropriate classes depending on the selected display (mobile, tablet, desktop)

PiotrJasinski1995 commented on src/partials/about.html:
```diff
@@ -42,24 +42,21 @@ <h3 class="about-list-header">04</h3>
-              ../img/memoji.png 1x,
-              ../img/memoji@2x.png 2x
-            "
+          <img class="about-picture-mobile"
```
look again on prop-leave-app section:
![prop-leave-app-img-html](https://github.com/user-attachments/assets/c420336e-d885-4950-aebe-3d769b0b9ca8)

PiotrJasinski1995 replied:
why do you add 3 different images? there is only one img

PiotrJasinski1995 commented on src/css/about.css:
```diff
@@ -148,6 +152,14 @@
         border-radius: 30px;
     }
+
+    .about-picture-mobile{
```
this styling is wrong, there is only one picture, use media queries with one class (there is only one image!) for proper styling for different screen sizes (please look again how it is done in prop-leave-app-section)

PiotrJasinski1995 replied:
what about ::after pseudo element display, it is also not corrected

PiotrJasinski1995 reviewed (changes requested):
please fix the errors

Jaciubek committed:
third commit

I removed 2 really unnecessary images, they are actually the same regardless of the device they are displayed on
